example/src/screens/RepliesScreen.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect, useState } from 'react';
+
+import { Replies } from 'instabug-reactnative';
+import { Heading } from 'native-base';
+
+import { ListTile } from '../components/ListTile';
+import { Screen } from '../components/Screen';
+
+export const RepliesScreen: React.FC = () => {
+  const [count, setCount] = useState<number>();
+
+  useEffect(() => {
+    Replies.getUnreadRepliesCount(setCount);
+  }, []);
```

Comment:
I think the example in the docs is totally different from what we are doing here. 
Our `useEffect` should be only called when the component mounts VS the one in the example which has a dependency on a state that can change after the component renders. So if the dependency kept on changing, they might lead to race-setting of it value. But I still find this irrelevant to what we do here.

Is there anything I am missing?

Reply:
As of React 18, even effects that should run only on mount do run twice in development with strict mode which is an intentional behavior the React team added to encourage developers to add cleanup functions to their effects to overcome problems like updating the state while the component is unmounted (the popular `Warning: Can't perform a React state update on an unmounted component`), race-setting of values (which can happen due to rendering the component twice).

I know we aren't on React 18 at the moment and that `Replies. getUnreadRepliesCount` does resolve in a tiny amount of time for it to be run after the component is unmounted, but I suggested it as a best practice to avoid problems with strict mode in the future.

Reply:
I think we should wait until React 18 migration, _which I do not see happening anytime soon_, instead of adding redundant code.
We should be targeting simplicity over future-proof best practices in our sample app, since it's just for demonstration and is not meant to be published.

Reply:
Alright, let's keep it as is for simplicity then.